Gems/AudioSystem/Code/Source/Engine/SoundCVars.cpp

```diff
@@ -132,7 +132,7 @@ namespace Audio::CVars
         "Usage: s_AudioObjectPoolSize=" AZ_TRAIT_AUDIOSYSTEM_AUDIO_OBJECT_POOL_SIZE_DEFAULT_TEXT "\n");
 
 #if defined (CARBONATED)
-    AZ_CVAR(float, s_PositionUpdateThreshold, 1.0f,
+    AZ_CVAR(float, s_PositionUpdateThreshold, 0.6f,
```

Comment:
That's still over half a meter :/  That's a LOT of distance.  Where/how are you calculating these distances?  Why not just use 0.1?